### ISSUE TITLE
improve poetry installation docs

### ISSUE THREAD

**Issue by boczekbartek** [RobotecAI/rai#286]
**Describe the bug**
When poetry is installed it is not added to PATH.It's installed in `~/.local/bin` and needs to be added manually.

**Comment by maciejmajek** [comment]
That's true. Though information about this is displayed during poetry installation. We can add a note for that

**Comment by maciejmajek** [comment]
Poetry prints info to add binary to path.
